source/builtins.rst

```diff
@@ -263,9 +262,9 @@ Python 2.6 and below didn't have an ``importlib`` module.
 
 If your code uses ``reload()``, import it conditionally on Python 3::
```

Comment:
I'm not sure if this still means "conditionally".

Reply:
Sure it does...  The code reads "if __reload__ is not already defined then import if from importlib".

Reply:
Thanks. I'm not a native speaker, so those thing are not always clear to me, but what you say makes sense.